src/App.jsx

```diff
@@ -56,7 +56,7 @@ export function App() {
 						}
 					/>
 					<Route path="/list" element={<List data={data} />} />
-					<Route path="/manage-list" element={<ManageList />} />
+					<Route path="/manage-list" element={<ManageList userId={userId} />} />
```

Comment:
I can see why you'd want to pass the `userId` prop to the `ManageList` component! Something to consider is that since `userId` is already accessible through the `useAuth` hook, it might be more efficient to retrieve it directly within `ManageList` rather than passing it down as a prop. This would reduce the need for additional props in the component tree and make `ManageList` more self-contained!

Reply:
i removed userId from props that are being down to ManageList component and extracted current userId from useAuth hook, great suggestion!